stance-4forums/README.md

```diff
@@ -0,0 +1,32 @@
+### Stance
+
+## Problem
+In this example, we attempt to identify the stance of a speaker in a debate as being for or against.
+
+## Dataset
+
+The dataset comes from two online debate websites: 4FORUMS.COM, from the Internet Argument Corpus and CREATEDEBATE.com.
```

Comment:
Add some more details about the contents of the data.
Mention how these datasets were collected.
Case these websites properly.